### ISSUE TITLE
sugestion: mention in README benefits over other FTS plugins

### ISSUE THREAD

**Issue by jonassmedegaard** [slusarz/dovecot-fts-flatcurve#15]
Hi,

I stumbled upon this project and am curious if beneficial to adopt for my own server and to include with Debian officially (where I am a developer).

After reading the nice and detailed README file, the one important question is still unanswered: How does it compare to other FTS plugins.
I was sorta expecting to get the answer in section `Why flatcurve` but understand that it is about "Why the name flatcurve" not "why make yet another plugin called flatcurve".
Then I noticed in `Acknowledgements` you mention conversations leading you to do the project - perhaps if those conversations were public and you could link to them my curiosity would be satisfied...

**Comment by slusarz** [comment]
See #4 - I think that addresses what you are asking for, and explains why this information is not in README.

That's cool that there is Debian interest in this.  I will note that the plans are to merge this soon into Dovecot core, although it is still unclear whether this will be done in 2.3 branch or the next branch (i.e. 2.4).  So it might be useful to package for the short-term, but long-term this won't be necessary as this will work with Dovecot CE out-of-the-box.

Closing this ticket as mostly a duplicate to #4.

**Comment by jonassmedegaard** [comment]
Indeed, this was already covered - my apologies for not checking that!

Sounds promising that your work will get integrated with Dovecot - Debian aims for long-term and I see no need for short-term packaging.

Thanks a lot!

**Comment by slusarz** [comment]
Makes sense to at least reference ticket #4 in the README, so others won't be confused like you.  Added in 3a0ac4a39c759142942cd1628eb6c6fb1f0934eb.

**Comment by paravoid** [comment]
Also a Debian Developer here - I built a Debian package for my own use and it works just fine and is standards-compliant enough to be included into Debian if we wanted that. I haven't filed an "intent to package" as -like Jonas- I'm not sure whether there is any point in packaging this if it were to be merged upstream soon. Is there a place I can read more about your upstreaming efforts or are these happening in private conversations?

Semi-relatedly, are you also planning to tag another version? I saw quite some progress compared to the last tagged release, so I opted into packaging a git snapshot.

**Comment by slusarz** [comment]
"Is there a place I can read more about your upstreaming efforts or are these happening in private conversations?"

These are internal discussions.  Realistically, it's looking like we won't be ready to release the next major version of OX Dovecot Pro until 2nd half of 2022, and next version bump of CE wouldn't happen until after that, so there probably will be a decent amount of time until that happens.

"Semi-relatedly, are you also planning to tag another version? I saw quite some progress compared to the last tagged release, so I opted into packaging a git snapshot."

I can tag a new version.  Need to resolve #16 before I do so, though.
